contrib/builder/rpm/generate.sh

```diff
@@ -22,7 +22,7 @@ for version in "${versions[@]}"; do
 	suite="${version##*-}"
 	from="${distro}:${suite}"
 	installer=yum
-	if [[ "$distro" == "fedora" ]] && [[ "$suite" -ge "22" ]]; then
+	if [[ "$distro" == "fedora" ]]; then
```

Comment:
not sure if all current CentOS and RHEL releases also have dnf installed by default now